onnxruntime/contrib_ops/cpu/transformers/beam_search_parameters.cc

```diff
@@ -31,15 +31,26 @@ void BeamSearchParameters::ParseFromInputs(OpKernelContext* context) {
   ORT_ENFORCE(context != nullptr);
   const Tensor* input_ids = context->Input<Tensor>(0);
   const auto& dims = input_ids->Shape().GetDims();
+  int initial_decode_sequence_length = 0;
   if (this->model_type == IGenerationParameters::kModelTypeWhisper) {
     ORT_ENFORCE(dims.size() == 3, "input_features shall have 3 dimensions. Got ", dims.size());
+    const Tensor* decoder_input_ids = context->Input<Tensor>(10);
+    const auto& decoder_dims = decoder_input_ids->Shape().GetDims();
```

Comment:
Another place that probably needs to be changed if `decoder_input_ids` isn't provided: should default to 1 if null value